File: MiniPrograms/C/Loops.c
#include <stdio.h>

int main()
{
    int loops, i;
    printf("How many loops loops loops would you like? ");
    scanf("%d", &loops);
    printf("I like");
    for (i = 0; i < loops; ++i)
    {
        printf(", loops");
    }
    printf(".");
    return 0;
}

File: Go/Loops.go
package main

import "fmt"

func main() {
	var loops int
	fmt.Println("How many loops loops loops would you like?")
	fmt.Scanln(&loops)
	fmt.Print("I like")
	for i := 0; i < loops; i++ {
		if i == 0 {
			fmt.Print(" loops")
		} else {
			fmt.Print(", loops")
		}
	}
	fmt.Println(".")
}
